Add meta description to some pages (#4534)
wazuh/wazuh-documentation: source/user-manual/capabilities/wazuh-logtest/index.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::

>   :description: The Wazuh-Logtest solution was designed to replace ossec-logtest. It allows to test and verify rules and decoders remotely. Learn how it works here. 

Reviewer: ```suggestion
  :description: The Wazuh-Logtest solution was designed to replace ossec-logtest. It allows you to test and verify rules and decoders remotely. Learn more about it here. 
```

>   :description: The Wazuh-Logtest whole solution was designed to replace ossec-logtest, now allowing to test and verify rules and decoders remotely. Learn more about it here. 

Reviewer 2: ```suggestion
  :description: The Wazuh-Logtest solution was designed to replace ossec-logtest. It allows to test and verify rules and decoders remotely. Learn how it works here. 
```
